src/Altinn.Platform/Altinn.Platform.Profile/Profile/Controllers/UsersController.cs

```diff
@@ -43,6 +43,23 @@ public async Task<ActionResult> Get(int userID)
             return Ok(result);
         }
 
+        /// <summary>
+        /// Gets the user profile for a given SSN
+        /// </summary>
+        /// <param name="ssn">The user's social security number</param>
+        /// <returns>User profile connected to given SSN </returns>
+        [HttpPost]
+        public async Task<ActionResult> Get([FromBody]string ssn)
```

Comment:
Syns kanskje det er litt rart at get tar id og post tar ssn og det er kun metoden som er forskjellig. Burde den hatt et mer RPC navn?  @SandGrainOne ?

Reply:
My initial thought was that the name `Get` could perhaps be a bit short, but I like that the names are the same. It is two operations performing work that have the same result, but with different input. Correct use of method overloading in C#.

I know remember that there is a separate issue here and that is the OpenAPI specification we generate. There the methods need to have different names as they are used to generate a method id in the spec.

Reply:
I could rename to GetUser(From/By/With)SSN ?

Reply:
I thinkt that is a good idea. We already break the rest principle with using post. Better make it clear it is a get, using post.